modules/general/external-dns/main.tf

```diff
@@ -22,7 +22,7 @@ resource "helm_release" "external_dns" {
   name       = "external-dns"
   repository = "https://kubernetes-sigs.github.io/external-dns"
   chart      = "external-dns"
-  version    = "1.10.1"
```

Comment:
There should not be a need for this version change?

Reply:
not necessary